### PR TITLE
Restructured

### PR DESCRIPTION
**Pull request by spyingcyclops**

### CODE REVIEW

File: vscode_client_2/README.md [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
+5. Under "host" write the URL of your mainframe connection in format http(s)://host(port) 
+
+### Create and Send your Message
+1. Create a new file in your VS Code workspace with the extension .txt
+2. Write your message in the file (Limit: 80 characters).
+3. Save the file
+4. In your workspace, right click .txt file and click on "send using Mainframe Messenger"
+5. A box appears at the top of the screen
+6. Write the mainframe username of the person you want to send the message to in the box. 
+**NOTE** The recipient must be logged in to the same mainframe server defined in your extension settings.
```
Comment (zeibura) [inline vscode_client_2/README.md:27]: You need two spaces at the end of the previous line for a soft return

File: vscode_client_2/README.md [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
-11. A box displays at the top of the screen
-12. Write the mainframe username of the person you want to send the message to in the box
-
-**NOTE** The person you send the message to must be logged in to the same mainframe server you put in extension settings
-
-13. Press enter to send your message
+Mainframe messenger is an easy way to send messages from Visual Studio Code to another mainframe user. 
+
+## Compatibility 
+Mainframe messenger has Zowe conformance and uses Zowe profile to connect to mainframe.
```
Comment (zeibura) [inline vscode_client_2/README.md:5]: Zowe profile is a prerequisite

File: vscode_client_2/README.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Mainframe Messenger for VS Code
+Mainframe messenger is an easy way to send messages from Visual Studio Code to another mainframe user.
```
Comment (DStatPersonal) [inline vscode_client_2/README.md:2]: Good and concise

Comment (spyingcyclops) [reply vscode_client_2/README.md:2]: thank you. Forgot to compliment your too :D
🗡️

**Pull request closed without merging**